bandit/core/utils.py

```diff
@@ -14,6 +14,21 @@
 
 LOG = logging.getLogger(__name__)
 
+HTTP_REQUEST_VERBS = {
```

Comment:
We don't typically put plugin specific constants into the core. We like to keep each plugin self contained. I'd rather see these constants back in the plugin(s) themselves, even if it would be duplicated amongst the two.

Reply:
I agree